SPEC.md

```diff
@@ -201,25 +201,170 @@ Note that logging mechanisms other than stdout and stderr are not required by th
 ### Isolators
 
 Isolators enforce resource constraints rather than namespacing.
-Isolators may be applied to individual applications, to whole containers, or to both.
+Isolators may be scoped to individual applications, to whole containers, or to both.
```

Comment:
We should be very explicit here about the required/desired behaviour of executors when a specified isolator cannot be fulfilled; I've been asked this a number of times now (even if it is sort of implied later on that they are optional). 

And especially so if we are introducing platform-specific isolators.